Source/Core/Core/Core.cpp

```diff
@@ -1046,7 +1047,8 @@ void UpdateInputGate(bool require_focus, bool require_full_focus)
   // Ignore full focus if we don't require basic focus
   const bool full_focus_passes =
```

Comment:
I realize someone else added this, but I was struggling to figure out what the heck the difference was between `require_focus` and `require_full_focus`.

Have you figured out what "full" and "non-full" focus are? :D